pkg/cmd/run/shared/shared.go

```diff
@@ -424,19 +425,31 @@ func preloadWorkflowNames(client *api.Client, repo ghrepo.Interface, runs []Run)
 }
 
 type JobsPayload struct {
-	Jobs []Job
+	TotalCount int `json:"total_count"`
+	Jobs       []Job
 }
 
 func GetJobs(client *api.Client, repo ghrepo.Interface, run *Run) ([]Job, error) {
 	if run.Jobs != nil {
 		return run.Jobs, nil
 	}
-	var result JobsPayload
-	if err := client.REST(repo.RepoHost(), "GET", run.JobsURL, nil, &result); err != nil {
-		return nil, err
+
+	v := url.Values{}
+	v.Set("per_page", "100")
```

Comment:
This is a minor nit but I thought it important to keep similar variable naming as other code:

```suggestion
	query := url.Values{}
	query.Set("per_page", "100")
```

Reply:
Good call. Thanks for pointing this out :)